.github/workflows/unit-tests.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: pip freeze
         run: pip freeze
       - name: Run unit tests
-        if: inputs.os == 'ubuntu-latest'
+        if: inputs.os == 'ubuntu-latest' || inputs.branch == 'develop'
```

Comment:
I see how this runs `make test` on `develop` 👍🏽 but I don't see how this prevents `make test-no-spark`? Does the block below require something like `&& inputs.branch != 'develop`?

Reply:
Yep I missed that! Added now